src/rpc/privatesend.cpp

```diff
@@ -36,14 +36,11 @@ UniValue privatesend(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_INTERNAL_ERROR, "Client-side mixing is not supported on masternodes");
 
     if (!privateSendClient.fEnablePrivateSend) {
-        if (fLiteMode) {
-            // mixing is disabled by default in lite mode
-            throw JSONRPCError(RPC_INTERNAL_ERROR, "Mixing is disabled in lite mode, use -enableprivatesend command line option to enable mixing again");
-        } else if (!gArgs.GetBoolArg("-enableprivatesend", true)) {
+        if (!gArgs.GetBoolArg("-enableprivatesend", true)) {
```

Comment:
code reading suggestion: use gArgs.GetBoolArg("-enableprivatesend", false) as this example:
https://github.com/dashpay/dash/blob/develop/src/wallet/wallet.cpp#L906

Reply:
this shouldn't be the case. That second argument is the default, what this line is saying is if "-enableprivatesend" isn't specified, assume that privatesend is enabled, so that someone would have to specifically disable privatesend with "-enableprivatesend=0"

Reply:
you are right, i thought that was comparing but the second argument is the default value indeed